core/Migrations/Version20211005081736.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace OC\Migrations;
+
+use OCP\IDBConnection;
+use OCP\Migration\ISqlMigration;
+
+/**
+ * Auto-generated migration step: Please modify to your needs!
+ */
+class Version20211005081736 implements ISqlMigration {
+
+	public function sql(IDBConnection $connection) {
+		$platform  = $connection->getDatabasePlatform();
+		$platformName = $platform->getName();
+
+		// Only execute this migration on mysql platform
```

Comment:
```suggestion
		// Only execute this migration on mysql platform (MySQL and MariaDb)
```

Maybe note here that this will apply to both MySQLcand MariaDb implementations of the "mysql" platform.